### PR TITLE
hotfix: Fix broken theme

### PR DESCRIPTION
**Pull request by Boot-Error**
Fix broken theme, issue with css not compiled properly.

### CODE REVIEW

**Comment by Boot-Error** [comment]
Any updates on this?

**Comment by AniVerma17** [comment]
Screenshot from 2021-07-11 20-50-05
@Boot-Error the site works good functionally.
However the layout and styles are missing on the webpage as seen in the screenshot (this is after building the project). Please apply them in a card-based layout. You can take reference to our current planet site.

File: src/_includes/partials/links.njk [hunk 1]
```diff
@@ -0,0 +1 @@
+<link rel="stylesheet" href="/assets/base.css">
```
Comment (AniVerma17) [inline src/_includes/partials/links.njk:1]: Please remove the leading slash from the CSS file path. This should potentially fix the theming issue.

**Review by AniVerma17 — CHANGES_REQUESTED** [review]
Screenshot from 2021-07-11 20-50-05
@Boot-Error the theming error is still persistent, as seen in the screenshot above (this is after running npm build).
Check the code review for the potential fix.

File: src/_includes/partials/links.njk (hunk 1 above)
Comment (Boot-Error) [reply src/_includes/partials/links.njk]: Removed the leading slash 20beecb89faa45b0afcbf7855930fbca4160cf98

**Review by AniVerma17 — APPROVED** [review]
CSS working as expected now, LGTM.

**Pull request merged**